atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -733,7 +733,8 @@ def check_manifest_list(self, build_image, orchestrator_platform, platforms,
 
         # orchestrator platform is in manifest list
         if orchestrator_platform in self.build_image_digests:
-            if self.build_image_digests[orchestrator_platform] != current_buildimage:
+            if '@sha256:' in current_buildimage and \
```

Comment:
Can we change these two checks around so it's a bit easier to understand?
```python
if orchestrator_platform not in self.build_image_digests:
  raise RuntimeError("Platform for orchestrator '%s' isn't in manifest list" 
                     % orchestrator_platform)

if ('@sha256:' in current_buildimag and 
    self.build_image_digests[orchestrator_platform] != current_buildimage):
  raise RuntimeError("Orchestrator is using image digest '%s' which isn't"
                     " in manifest list" % current_buildimage)
```

Note that we should probably change the error message to specify that orchestrator is using image digest.